Assets/Tests/AlgorithmTests.cs

```diff
@@ -0,0 +1,96 @@
+using System;
+using System.Collections.Generic;
+using Enums;
+using LogicServices;
+using LogicServices.Algorithm;
+using Models;
+using NUnit.Framework;
+
+namespace Tests
+{
+    public class AlgorithmTests
+    {
+        // A Test behaves as an ordinary method
+        [Test]
+        public void CalculateRandomness_HighRandomness_ReturnsTrue()
+        {
+            //Given
+            Randomness randomness = new Randomness(1);
+            
+            //Then
+            Assert.IsTrue(randomness.Calculate());
+        }
+        
+        [Test]
+        public void CalculateRandomness_LowRandomness_ReturnsFalse()
+        {
+            //Given
+            Randomness randomness = new Randomness(0);
+            
+            //Then
+            Assert.IsFalse(randomness.Calculate());
```

Comment:
Always use variables in asserts not methods. Makes debugging easier when needed.